sections/errorhandling/shuttingtheprocess.polish.md

```diff
@@ -0,0 +1,99 @@
+# Opuść ten proces z wdziękiem, gdy do miasta przybywa nieznajomy
+
+### Wyjaśnienie jednego akapitu
```

Comment:
"### Wyjaśnienie jednego akapitu" maybe change it to: "Wyjaśnienie w jednym akapicie"

Reply:
Agree, fixed. Done, thanks a lot!